pkg/cmd/roachprod/flags.go

```diff
@@ -330,4 +331,9 @@ Default is "RECURRING '*/15 * * * *' FULL BACKUP '@hourly' WITH SCHEDULE OPTIONS
 		cmd.Flags().BoolVar(&secure,
 			"secure", false, "use a secure cluster")
 	}
+	for _, cmd := range []*cobra.Command{pgurlCmd, sqlCmd} {
+		cmd.Flags().StringVar(&tenantName,
+			"tenant-name", "", "specific tenant to connect to")
```

Comment:
specific _app_ tenant, right? The user connects to the system tenant by default, correct?

Reply:
By default the user does connect to the system tenant. But, for the purposes of creating a connection, the system tenant isn't too special. If you pass "system" to the --tenant-name flag you will connect to the system tenant.